components/email/src/Email.svelte

```diff
@@ -650,14 +650,14 @@
     }
   }
 
```

Comment:
`fetchIndividualMessage` function is updated to return the message itself, instead of just the body property.

Minor change here, but to be consistent with Mailbox and will better suits its name and purpose.